linkerd/examples/mutualTls.yaml

```diff
@@ -7,6 +7,15 @@ routers:
 - protocol: http
   dtab: |
     /svc => /#/io.l5d.fs
+  client:
+    tls:
+      commonName: foo
+      trustCerts:
+      - /foo/cacert.pem
+      clientAuth:
+        certPath: /foo/clientCert.pem
+        keyPath: /foo/clientKey.pem
+        clientCaChainPath: /foo/intermediateCa.pem
```

Comment:
Same thing here. It seems like `client` in the name is inconsistent with the other parts of the `clientAuth` setting.